include/envoy/http/header_map.h

```diff
@@ -273,6 +273,8 @@ class HeaderEntry {
   HEADER_FUNC(EnvoyInternalRequest)                                                                \
   HEADER_FUNC(EnvoyIpTags)                                                                         \
   HEADER_FUNC(EnvoyMaxRetries)                                                                     \
+  HEADER_FUNC(EnvoyRateLimitedResetHeaders)                                                        \
+  HEADER_FUNC(EnvoyRateLimitedResetMaxIntervalMs)                                                  \
```

Comment:
I'm not crazy about adding new inline headers here that very few people will use. See my comments above.

Reply:
Roger that, I'll remove them.